src/utils/validateEnvironment.ts

```diff
@@ -1,9 +1,13 @@
 import responseMessages from '../constants/responseMessages';
+import * as process from 'process';
 
 const { API_KEY_NOT_FOUND } = responseMessages;
 
 export default function validateEnvironment(): void {
-  if (!process.env.RATE_SOURCE_API_KEY) {
+  if (
+    !process.env.CURRENCY_BEACON_API_KEY ||
+    !process.env.CURRENCY_API_API_KEY
```

Comment:
Maybe we shouldn't throw error because of this since we still have providers that don't require an API key